sound/soc/sof/ipc4-topology.h

```diff
@@ -401,6 +401,24 @@ struct sof_ipc4_src {
 	struct sof_ipc4_msg msg;
 };
 
+/**
+ * struct sof_ipc4_asrc ASRC config data
+ * @base_config: IPC base config data
+ * @sink_rate: Output rate for sink module
+ * @asynchronous_mode: synchronous 0, asynchronous 1.
```

Comment:
what does this mean?

Reply:
In synchronous mode ASRC operates exactly like SRC with fixed fraction like 32/48.

In asynchronous mode the skew is measured and the ASRC converts with fraction e.g.  skew * 32 / 48.

Reply:
why would we want to use the synchronous mode when? We can use an SRC module instance which presumably has better audio performance, no?

Reply:
I added synchronous mode because it allowed to test ASRC with DAI endpoint without timestamps capability, e.g. testbench file I/O. It still runs the same algorithm core so testing possibility has value. The synchronous mode comes for free since it only disables a part of algorithm without adding anything. It's also legacy from IPC3 and TPLG1.

Reply:
I think this is worthy of a comment, because an asynchronous mode with a value of 'synchronous' will be difficult to parse for most reviewers. It's a self-negating propostion...